onert-micro/CMakeLists.txt

```
@@ -118,3 +118,56 @@ add_custom_command(
 )
 
 add_custom_target(luci_interpreter_micro_arm DEPENDS "${MICRO_ARM_BINARY}")
+
```

What do you think about using an additional flag so that mbed os for qemu is not always going to build?
Maybe smth like: `BUILD_TEST`, and if this flag is declared then we will build test infrastructure?

```suggestion
if (NOT DEFINED BUILD_TEST)
    return()
endif()
```